XRTK.WebXR/Packages/com.xrtk.webxr/package.json

```diff
@@ -10,18 +10,17 @@
     "mixed",
     "reality"
   ],
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
package version should be `0.2.0`

Reply:
I've actually reset this to 0.1.0, as it is the first iteration for the WebXR package.  Dependencies shouldn't need to follow the minor version of its parent.  Thoughts @StephenHodgson ?